daemon/plugin-fuzz/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "ethersync-plugin-fuzz"
+description = "Fuzzing the daemon via simulated plugin input over the socket."
```

Comment:
The fuzzer is actually not simulating the plugins, it's running the real deal, booting up two `Daemon` instances, and spawning two `nvim`s :) Think the description could be more generic, allowing us to add other integration-y tests later.

Reply:
I said end-to-end testing for clients! Let me know what you think.